04/app.js

```diff
@@ -1,9 +1,33 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+	firstName: "Adam",
+	lastName: "Nowak",
+	born: {
+		day: "13",
+		month: "04",
+		year: "1985",
+	},
+};
+
+const today = new Date();
+console.log(today);
+
+const currentDay = today.getDate();
+console.log(currentDay);
+
+const currentMonth = today.getMonth() + 1;
+console.log(currentMonth);
+
+const birthday = function (object) {
```

Comment:
Jest to "anonimowe wyrażenie funkcyjne" z `const`, a dla `cost` nie działa "hoisting".
Natomiast jak zamienimy na `var` to też nie zadziała, bo w przypadku "anonimowego wyrażenia funkcyjnego ", "hoisting" przerzuci tylko `var birthday` na górę co powoduje, że wtedy mamy `birthday = undefined` co nie da się wywołać. Daj znać na Slacku czy to jasne dla Ciebie ;)

Reply:
tak :) po zrobieniu zadań na to wpadłam ;) i jak zamieniłam birthday na anonimową funkcję a nie wyrażenie, to hoisting zadziałał ;)